src/Form/UserEditType.php

```diff
@@ -43,10 +58,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('email', EmailType::class, [
                 'label' => 'label.email',
             ])
-            ->add('enabled', YesNoType::class, [
-                'label' => 'label.active',
-            ])
         ;
+
+        $auth = $this->security;
+        if ($auth->isGranted('system_configuration')) {
```

Comment:
Instead of checking the permission, please add a flag in `configureOptions()`:
```
        $resolver->setDefaults([
            'include_active_flag' => false,
```
and here:
```
if ($options['include_active_flag']) {
...
```
The new constructor can be removed completely

Reply:
Done